docs/src/tips_for_packages.md

```diff
@@ -0,0 +1,43 @@
+# Tips for making your package work with AD
+
+## Ignoring gradients for certain expressions
+
+There exists code that is not meant to be differentiated through, for example logging.
+In some cases, AD systems might work perfectly well with that code, but in others they might not.
+A convenience function `ignore_gradients` is provided to get around this issue.
+It captures the functionality of both `Zygote.ignore` and `Zygote.dropgrad`
+
+For example, Zygote will break if you try to store intermediate values like so because it does not support mutation.
```

Comment:
```suggestion
For example, Zygote does not support mutation, so it will break if you try to store intermediate values as in the following example:
```